tests/test_packages/test_protocols/test_register.py

```diff
@@ -0,0 +1,177 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2022 Valory AG
+#   Copyright 2018-2020 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
```

Comment:
I suggest you throw this file away and then write a new one from scratch. Good exercise!